Azure/cc-setup-azure.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description: This script will make the next flow
+#
+# 1) Create a group in azure
+# 2) Create a VM
+# 3) Create a disk
+# 4) Attach previous disk to VM
+# 5) Set up Clear containers stuff.
```

Comment:
`Set up Clear containers components`

Reply:
agree - for all occurrences s/stuff/components/ or s/stuff/items/ pls :-)